### PR TITLE
add instructions for creating a new project.

### PR DESCRIPTION
**Pull request by Muhammad-Owais-Warsi**
Added the get started guide and link to the docs for the respective frameworks.

### CODE REVIEW

File: src/content/docs/reference/nodejs.mdx [hunk 1]
```diff
@@ -23,7 +23,7 @@ import ClientOverrideTS from "/src/snippets/reference/nodejs/ClientOverride.ts?r
 import ClientOverrideJS from "/src/snippets/reference/nodejs/ClientOverride.js?raw";
 import NodeVersions from "/src/snippets/reference/node-versions.mdx";
 import Comments from "/src/components/Comments.astro";
-import Requirements from "@/snippets/get-started/node-js/Requirements.mdx";
+import Requirements from "@/snippets/get-started/node-js-express/node-js/Requirements.mdx";
```
Comment (davidmytton) [inline src/content/docs/reference/nodejs.mdx:26]: I think this is an error? That file doesn't exist.

File: src/snippets/get-started/node-js/Step0.mdx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{/* prettier-ignore */}
+
+To install Node.js on your system follow the official ([docs](https://nodejs.org/en/download/package-manager)).
```
Comment (davidmytton) [inline src/snippets/get-started/node-js/Step0.mdx:3]:
```suggestion
To install Node.js on your system follow the official [docs](https://nodejs.org/en/download/package-manager).
```

File: src/snippets/get-started/deno/Step0.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+```sh
+curl -fsSL https://deno.land/install.sh | sh
+```
+2. Windows
+```sh
+irm https://deno.land/install.ps1 | iex
+
+```
+
+For detailed instructions refer to the Deno ([docs](https://docs.deno.com/runtime/getting_started/installation)).
```
Comment (davidmytton) [inline src/snippets/get-started/deno/Step0.mdx:15]:
```suggestion
For detailed instructions refer to the [Deno docs](https://docs.deno.com/runtime/getting_started/installation).
```

File: src/snippets/get-started/nest-js/Step0.mdx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{/* prettier-ignore */}
+
+Installing Nest.js
+
+```sh
+npm i -g @nestjs/cli
+```
+For detailed instructions refer to the Nest.js ([docs](https://docs.nestjs.com/)).
```
Comment (davidmytton) [inline src/snippets/get-started/nest-js/Step0.mdx:8]:
```suggestion
For detailed instructions refer to the [Nest.js docs](https://docs.nestjs.com/).
```

File: src/snippets/get-started/next-js/Step0.mdx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{/* prettier-ignore */}
+
+To create a Next.js project:
+
+```sh
+npx create-next-app@latest
+```
+For detailed instructions refer to the Next.js ([docs](https://nextjs.org/docs/app/getting-started/installation)).
```
Comment (davidmytton) [inline src/snippets/get-started/next-js/Step0.mdx:8]:
```suggestion
For detailed instructions refer to the [Next.js docs](https://nextjs.org/docs/app/getting-started/installation).
```

File: src/snippets/get-started/node-js-express/Step0.mdx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{/* prettier-ignore */}
+
+To install Express: 
+
+```sh
+npm install express
+```
+For detailed instructions refer to Express.js ([docs](https://expressjs.com/en/starter/installing.html)).
```
Comment (davidmytton) [inline src/snippets/get-started/node-js-express/Step0.mdx:8]:
```suggestion
For detailed instructions refer to [Express.js docs](https://expressjs.com/en/starter/installing.html).
```

File: src/snippets/get-started/node-js-hono/Step0.mdx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+<div slot="yarn" slotIdx="3">
+
+```sh
+yarn create hono
+```
+
+</div>
+</SelectableContent>
+
+For detailed instructions refer to the Hono ([docs](https://hono.dev/docs/#quick-start)).
```
Comment (davidmytton) [inline src/snippets/get-started/node-js-hono/Step0.mdx:31]:
```suggestion
For detailed instructions refer to the [Hono docs](https://hono.dev/docs/#quick-start).
```

File: src/snippets/get-started/node-js-hono/Step0.mdx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+To install Node.js on your system follow the official ([docs](https://nodejs.org/en/download/package-manager)).
```
Comment (davidmytton) [inline src/snippets/get-started/node-js-hono/Step0.mdx:33]:
```suggestion
To install Node.js on your system follow the official [docs](https://nodejs.org/en/download/package-manager).
```

File: src/snippets/get-started/remix/Step0.mdx [hunk 9]
```diff
@@ -0,0 +1,9 @@
+{/* prettier-ignore */}
+
+To create a Remix project:
+
+```sh
+npx create-remix@latest
+```
+For detailed instructions refer to the Remix ([docs](https://remix.run/docs/en/main/start/quickstart#installation)).
```
Comment (davidmytton) [inline src/snippets/get-started/remix/Step0.mdx:8]:
```suggestion
For detailed instructions refer to the [Remix [docs](https://remix.run/docs/en/main/start/quickstart#installation).
```

File: src/snippets/get-started/sveltekit/Step0.mdx [hunk 10]
```diff
@@ -0,0 +1,9 @@
+{/* prettier-ignore */}
+
+To create a SvelteKit project:
+
+```sh
+npx sv create my-app
+```
+For detailed instructions refer to the SvelteKit ([docs](https://svelte.dev/docs/kit/creating-a-project)).
```
Comment (davidmytton) [inline src/snippets/get-started/sveltekit/Step0.mdx:8]:
```suggestion
For detailed instructions refer to the [SvelteKit docs](https://svelte.dev/docs/kit/creating-a-project).
```

**Review by davidmytton — CHANGES_REQUESTED** [review]
Looks good. A few minor changes.

**Comment by Muhammad-Owais-Warsi** [comment]
Sure, I'll make the changes.

**Comment by davidmytton** [comment]
Great, thanks for working on this!

**Pull request merged**